spec/features/saml/ial2_sso_spec.rb

```diff
@@ -90,7 +90,7 @@ def sign_out_user
       )
     end
 
-    context 'having previously selected USPS verification' do
+    context 'having previously selected USPS verification', js: true do
```

Comment:
I'm very curious how our test suite speed will change with all these `js: true` being added, since it seems like that's the slower driver for me. Maybe after this PR lands we should re-run knapsack weights? (or maybe include in this PR?)

Reply:
> I'm very curious how our test suite speed will change with all these `js: true` being added, since it seems like that's the slower driver for me. Maybe after this PR lands we should re-run knapsack weights? (or maybe include in this PR?)

Good point on the Knapsack report. I might do that as a quick follow-on since this is already a pretty noisy diff.

You might also be interested in the recap notes from the engineering sync you missed last week, which touched on the topic of turning on the JavaScript driver for more of these specs.